test/test_services_constructor.jl

```diff
@@ -586,3 +586,164 @@ end
+
+    results = OptimizationProblemResults(ps_model)
+    interarea_flow = read_variable(results, "FlowActivePowerVariable__AreaInterchange")
+    # The values for these tests come from the data
+    @test all(interarea_flow[!, "1_2"] .<= 150)
+    @test all(interarea_flow[!, "1_2"] .>= -150)
+
+    load = read_parameter(results, "ActivePowerTimeSeriesParameter__PowerLoad")
+    thermal_gen = read_variable(results, "ActivePowerVariable__ThermalStandard")
+
+    zone_1_load = sum(eachcol(load[!, ["Bus4_1", "Bus3_1", "Bus2_1"]]))
+    zone_1_gen = sum(
+        eachcol(
+            thermal_gen[
+                !,
+                ["Solitude_1", "Park City_1", "Sundance_1", "Brighton_1", "Alta_1"],
+            ],
+        ),
+    )
+    @test all(
+        isapprox.(
+            sum(zone_1_gen .+ zone_1_load .- interarea_flow[!, "1_2"]; dims = 2),
+            0.0;
+            atol = 1e-3,
+        ),
+    )
+
+    zone_2_load = sum(eachcol(load[!, ["Bus4_2", "Bus3_2", "Bus2_2"]]))
+    zone_2_gen = sum(
+        eachcol(
+            thermal_gen[
+                !,
+                ["Solitude_2", "Park City_2", "Sundance_2", "Brighton_2", "Alta_2"],
+            ],
+        ),
+    )
+    @test all(
+        isapprox.(
+            sum(zone_2_gen .+ zone_2_load .+ interarea_flow[!, "1_2"]; dims = 2),
+            0.0;
+            atol = 1e-3,
+        ),
+    )
+end
+
+@testset "Test Interfaces on Interchanges" begin
+    sys_rts_da = build_system(PSISystems, "modified_RTS_GMLC_DA_sys")
+    transform_single_time_series!(sys_rts_da, Hour(24), Hour(1))
+    interchange1 = AreaInterchange(;
+        name = "interchange1_2",
+        available = true,
+        active_power_flow = 100.0,
+        flow_limits = (from_to = 1.0, to_from = 1.0),
+        from_area = get_component(Area, sys_rts_da, "1"),
+        to_area = get_component(Area, sys_rts_da, "2"),
+    )
+    interchange2 = AreaInterchange(;
+        name = "interchange1_3",
+        available = true,
+        active_power_flow = 100.0,
+        flow_limits = (from_to = 1.0, to_from = 1.0),
+        from_area = get_component(Area, sys_rts_da, "1"),
+        to_area = get_component(Area, sys_rts_da, "3"),
+    )
+    interchange3 = AreaInterchange(;
+        name = "interchange3_2",
+        available = true,
+        active_power_flow = 100.0,
+        flow_limits = (from_to = 1.0, to_from = 1.0),
+        from_area = get_component(Area, sys_rts_da, "3"),
+        to_area = get_component(Area, sys_rts_da, "2"),
+    )
+    add_components!(
+        sys_rts_da,
+        [interchange1, interchange2, interchange3],
+    )
+    # This interface is limiting all the flows into 1
+    interface = TransmissionInterface(;
+        name = "interface1_2_3",
+        available = true,
+        active_power_flow_limits = (min = 0.0, max = 1.0),
+        violation_penalty = 1000.0,
+        direction_mapping = Dict("interchange1_2" => 1,
+            "interchange1_3" => -1,
+        ),
+    )
```

Comment:
interchange1_2 and interchange1_3 are both defined as the flows out of area 1, so wouldn't the direction be -1 for both of them when defining the interface if we want to limit all the flows into 1?